### ISSUE TITLE
"libavcodec58 not found on your system" [-22] error

### ISSUE THREAD

**Issue by CPioGH2002** [flathub/com.parsecgaming.parsec#39]
Hi there. Great software so far, I'm a fan. :-)

However, using it on **Linux (Kubuntu 23.04)** via **Flatpak** (_com.parsecgaming.parsec stable_) yields the above error message as of late. It did work fine not too long ago though. Naturally, one tries to install libavcodec58 to resolve the issue but since libavcodec5**9** is the current release, installing the older one either is impossible, fails or mixes up the rest of the system without solving anything.

I did try the **.deb package** too, getting rid of the Flatpak release, but this, as you may have guessed, hangs on the same issue:

> dpkg: dependency problems prevent configuration of parsec:
>  parsec depends on libavcodec58; however:
>   Package libavcodec58 is not installed.

So us crazy Linux people may need a way to have Parsec running with the libavcodec5**9** package to eventually be able to use your great streaming solution again.

vvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvv
**Edit:**
My older **Linux Mint** machine, which still uses libavcodec58, does work as expected, so the recent update of the libavcodec package/library may have caused this. Means it's not exactly Parsec's problem but I would expect the Flatpak installation to be able to bring everything needed to eventually run on any system. Wrong thinking? 

^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

In short:
The dependency on libavcodec58 (as opposed to the current libavcodec5**9** release) seems to be the issue.

Workarounds:
- None -

Wishes:
Sure, if you'd asked me for extra wishes, a proper Parsec release which allows streaming _from_ Linux would also be appreciated. :-)

**Comment by BentHaase** [comment]
Same here, I guess the latest commit breaks things (d14fd7202f48211d09f23ba60510d3ef6726cbfc, as well as 012802b2bbce64bc595659866a6438e621cf681e). Same issue as discussed in #23.

![image](https://github.com/flathub/com.parsecgaming.parsec/assets/5788982/a5756ef2-2257-4e37-8501-2fb095051da8)

**Comment by BigDemarest** [comment]
Getting the same error here, running Parsec via Flatpak on the Discover store on Steam Deck in desktop mode

**Comment by SISheogorath** [comment]
I pushed an rollback earlier, it should roll out within the next 3-4 hours. This will mean, we run on an unmaintained runtime, but that's not really worse than having a broken setup in first place.

It probably ruined one or another game night, and I'm sorry for that. I'm currently thinking how we can get the beta channel working so people feeling a bit more comfortable to roll things back when they go south, while also maintaining a stable built for everyone else.

If you want to do something yourself, please get in contact with upstream parsec and ask for a built against a modern version of libav.

**Comment by BentHaase** [comment]
> I pushed an rollback earlier, it should roll out within the next 3-4 hours. This will mean, we run on an unmaintained runtime, but that's not really worse than having a broken setup in first place.
> 
> It probably ruined one or another game night, and I'm sorry for that. I'm currently thinking how we can get the beta channel working so people feeling a bit more comfortable to roll things back when they go south, while also maintaining a stable built for everyone else.
> 
> If you want to do something yourself, please get in contact with upstream parsec and ask for a built against a modern version of libav.

Thanks for your immediate efforts, no need to apologize :slightly_smiling_face:

**Comment by BentHaase** [comment]
Preliminary feedback: When pulling 0d5be11bbb6285cc0b14220e54df7d568048d18b and building locally I get a working Parsec and can connect again. :heavy_check_mark:
